src/_P008_RFID.ino

```diff
@@ -113,18 +113,16 @@ boolean Plugin_008(byte function, struct EventStruct *event, String& string)
               }
               break;
             }
-            // reset everything
-            unsigned long bitCount = Plugin_008_bitCount;    // copy for log
-            unsigned long keyBuffer = Plugin_008_keyBuffer;  // copy for log
-            Plugin_008_keyBuffer = 0;
-            Plugin_008_bitCount = 0;
-            Plugin_008_timeoutCount = 0;
             // write log
             String log = F("RFID : Tag: ");
-            log += keyBuffer;
+            log += (unsigned long) Plugin_008_keyBuffer;
             log += F(" Bits: ");
-            log += bitCount;
+            log += Plugin_008_bitCount;
             addLog(LOG_LEVEL_INFO, log);
```

Comment:
Please wrap in check for loglevel active.

Reply:
all requested changes +1 committed :)